setup.py

```diff
@@ -15,7 +15,7 @@
     "pysam"
 ]
 
-VERSION = "4.2.0"
+VERSION = "4.3.0-SNAPSHOT"
```

Comment:
goooood. I don't like at all but we have to keep versions synchronised both in `setup.py` and `pom.xml`